tests/gis_tests/geoapp/test_functions.py

```diff
@@ -58,9 +59,12 @@ def test_asgeojson(self):
         # SELECT ST_AsGeoJson("geoapp_city"."point", 8, 2) FROM "geoapp_city"
         # WHERE "geoapp_city"."name" = 'Houston';
         # This time we want to include the CRS by using the `crs` keyword.
+        if mysql:
+            houston_json = json.loads(houston_json)
+            del houston_json['crs']
         self.assertJSONEqual(
+            City.objects.annotate(json=functions.AsGeoJSON('point', crs=True)).get(name='Houston').json,
```

Comment:
It's required as `assertJSONEqual` accepts only strings as the first argument. The other issues are fixed. Thanks!